### ISSUE TITLE
minor documentation correction in howto/components.html

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2655]
|<img alt="zirpu's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| zirpu reported|
|-|-|
|Trac ID|trac#2655|
|Type|defect|
|Created|2007-05-16 17:45:35Z|

http://twistedmatrix.com/projects/core/documentation/howto/components.html

I brought this up in irc, and radix pointed out it's a leftover
from previous implementations.

The bug is after the 2nd 'class AmericanSocket:' definition.
the paragraph following that has the following extraneous sentence:

A single item tuple in Python is created by enclosing an item in parentheses and placing a single trailing comma after it.

<details><summary>Searchable metadata</summary>

```
trac-id__2655 2655
type__defect defect
reporter__zirpu zirpu
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1179337535000000 1179337535000000
changetime__1179496094000000 1179496094000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun set status to `closed`|
|-|-|

(In [20297]) Remove extraneous Python syntax tip from interface/component documentation

Author: zirpu
Reviewer: exarkun
Fixes [#2655](https://github.com/twisted/twisted/issues/2655)

The construction of tuples is no longer relevant to the declaration of
interfaces, so drop a tip about how to construct a one-element tuple.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
